service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/CreateAndDeleteAzureControlledResourceFlightTest.java

```diff
@@ -1,5 +1,7 @@
 package bio.terra.workspace.service.resource.controlled.cloud.azure;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I think the WSM style guide says no `*` imports. I have the following in my IDE:
<img width="982" alt="image" src="https://user-images.githubusercontent.com/5368863/171242238-86211942-cd38-447a-af16-80c53e636c9f.png">

Reply:
I have another PR which I am currently working. Will fix this problem there.